config.json.example

```diff
@@ -0,0 +1,19 @@
+{
+    "urls": {
+        "Rest service": {
+            "url": "https://example.com/app/to/check",
+            "interval": 60000
```

Comment:
It'd make easier if the optional keys were preset here (as comments with an example value)

Reply:
Hmmmm okay, I thought it was clear enough having two different URLs, one with all keys and one with only the necessary ones